news/2018-03-13-new-mappers-guild-pack-release.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "New Mappers' Guild Pack Now Available"
+date: 2018-03-13 05:00:00 +0000
+---
+
+Straight from some of the Guild's finest mappers, we're proud to present a brand new beatmap pack and shiny new medal to accompany out! Check out some of the latest maps set to some of osu!'s Featured Artists!
```

Comment:
`shiny new medal to accompany out!` =>  
`a shiny new medal to accompany it!` ?